addons/l10n_ar/views/report_invoice.xml

```diff
@@ -16,7 +16,7 @@
                         <span style="font-size: x-small;" t-esc="not pre_printed_report and document_legend or '&#160;'"/>
                     </span>
                 </div>
-                <div name="right-upper-side" class="col-5 text-right" t-if="not pre_printed_report">
+                <div name="right-upper-side" class="col-5 text-right" style="padding-left: 0px;" t-if="not pre_printed_report">
```

Comment:
@Bruno-Zanotti What does this have to do with it?

Reply:
This is because of the lenght of the new document type.

Without this change:
[LP-B 00001-00000003.pdf](https://github.com/odoo/odoo/files/6067972/LP-B.00001-00000003.pdf)

With this change:
[LP-A 00001-00000002.pdf](https://github.com/odoo/odoo/files/6067976/LP-A.00001-00000002.pdf)

Reply:
ok, makes sense